Dockerfile

```diff
@@ -21,7 +21,7 @@ COPY ./poetry.lock ./pyproject.toml ./
 RUN poetry install --no-dev --no-root
 
 # `production` stage uses the dependencies downloaded in the `base` stage
-FROM python:3.10.5-slim as production
```

Comment:
Hmm, I'm surprised dependabot clobbered the `-slim` off. Seems like an oversight 🤔